test/reversible_tok.jl

```diff
@@ -0,0 +1,26 @@
+using Test
+using WordTokenizers
+
```

Comment:
use named `@testset`s rather than comment to make it clear what these tests each do